drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_services_loader.rb

```diff
@@ -202,6 +202,7 @@ def row_enrollment_id(row)
           row_value = row.field_value(ANSWER)
           answer = row_value ? element[:cleaner].call(row_value) : nil
           Array.wrap(answer).each do |value|
+            # FIXME we should set field_type for some of these
             cdes << cde_helper.new_cde_record(value: value, owner_type: service_class.name, owner_id: service.id, definition_key: key)
```

Comment:
outstanding item. we are storing some boolean questions in string types which might cause issues

Reply:
If the loader has to create a new cded, it looks like it will default to string type. Then the cde importer uses that definition field type to set the appropriate value field. So correcting the cded field type and running the importer seems like it should fix the issue